share/completions/fab.fish

```diff
@@ -0,0 +1,75 @@
+# tasks 
+for task in (fab -l)
```

Comment:
This should be put into a function that is then called via `complete -c fab -a '(thatfunction)'`.

Otherwise these won't change if anything changes about the output (e.g. because you've switched to using a different file).